### ISSUE TITLE
Add Frame Number and SYN/ACK Flags

### ISSUE THREAD

**Issue by cgivre** [mapr-demos/drill-pcap-format#17]
Hello all, 
Would it be of interest to add the frame number to the possible results?  I was also thinking it might be useful to have the possibility of returning a map of the SYN/ACK flags.  
If there's interest I can take a stab at that.
-- C

**Comment by tdunning** [comment]
Yes. Those are both useful.  I would recommend adding both "flags" as well
as "flag_map" so the user could pick a representation. With lazy eval
having both is fine.
